docs/overview_and_guide.rst

```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
 An Ethereum client is required in both cases. The Raiden binary takes the same command line
 arguments as the ``raiden`` script.
 
+Raspberry Pi
+************
+
+Download the latest :code:`raiden-<version>-linux-armv7l.tar.gz` for the Raspberry Pi Model 2B or
```

Comment:
Hey @eorituz it looks good but perhaps add a link to the releases page where these binaries can be found here?

Reply:
Thanks.
Not sure if I understand your point correctly but:
The Link to the releases page is in the genereal [`installation section`](https://raiden-network.readthedocs.io/en/latest/overview_and_guide.html#installation) right above the "individual OS/architecture installation guides"
I would keep it the way it is tbh

Reply:
Aha yes you are right. If it's right on the above section there is no need for a link here.